shared/util/feature-flags.js

```diff
@@ -5,23 +5,31 @@ import getenv from 'getenv'
 // To enable a feature, include it in the environment variable KEYBASE_FEATURES.
 // For example, KEYBASE_FEATURES=tracker2,login,awesomefeature
 
+const tracker2Key = 'tracker2'
+const loginKey = 'login'
+const mobileAppsExistKey = 'mobileAppsExist'
+
 type FeatureFlags = {
-  tracker2: boolean,
-  login: boolean
+  'tracker2': boolean,
+  'login': boolean,
+  'mobileAppsExist': boolean
 }
 
 let features = getenv.array('KEYBASE_FEATURES', 'string', '')
 
 // For compatibility, this is deprecated
-if (getenv.boolish('KEYBASE_TRACKER_V2', false)) { features.push('tracker2') }
-if (getenv.boolish('KEYBASE_ALLOW_LOGIN', false)) { features.push('login') }
+if (getenv.boolish('KEYBASE_TRACKER_V2', false)) { features.push(tracker2Key) }
+if (getenv.boolish('KEYBASE_ALLOW_LOGIN', false)) { features.push(loginKey) }
+if (getenv.boolish('KEYBASE_MOBILE_APPS_EXIST', false)) { features.push(mobileAppsExistKey) }
```

Comment:
If these are deprecated, maybe we shouldn't add more.  You can use KEYBASE_FEATURES_MOBILE_APPS_EXIST instead, is that okay?